jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -81,3 +81,81 @@ The OpenID Connect ID token also contains some https://openid.net/specs/openid-c
 | `oidc.circleci.com/context-ids`
 | An array of strings containing UUIDs that identify the context(s) used in the job. Currently, just one context is supported.
 |===
+
+
+==  Authenticate jobs with cloud providers
```

Comment:
Could we say instead "Authenticate with AWS from within a job" since AWS is the only cloud provider currently addressed?